qas/esstore/es_connect.py

```diff
@@ -149,7 +152,6 @@ def update_index(self, current_version):
 
         if updated_mapping is not None:
             self.__es_conn__.indices.close(index=__index_name__)
-            res = self.__es_conn__.indices.put_mapping(index=__index_name__, doc_type=__doc_type__, body=updated_mapping)
```

Comment:
@idoroiengel don't remove this snippet. This is used to update an index mapping of elasticsearch index. So, when we change the index mapping it has to be updated.

Reply:
Ok, how do I do this inside the pull request?

Reply:
You can just make the change a do another commit and it will show up in this PR.